packages/api/src/middleware/acl/checkGoogleJwt.js

```diff
@@ -9,6 +9,18 @@ async function checkGoogleJwt(req, res, next) {
   }
   const token = authorization.replace('Bearer ', '');
 
+  // For implicit flow from @react-oauth useGoogleLogin() hook from the email invite view
+  if (req.body.isAccessToken) {
+    try {
+      const tokenInfo = await client.getTokenInfo(token);
+      req.user = { email: tokenInfo.email, sub: tokenInfo.sub };
+      return next();
+    } catch (error) {
+      console.error(error);
+    }
+  }
+
+  // Original flow & Sign in with Google Button on main login page
```

Comment:
Are we using the original flow anymore? (Can we delete it?)

Reply:
Yes, this is used by the main login page (the one that now has the new personalized Sign in with Google button). I will take "Original flow" out of that comment!